### ISSUE TITLE
GC-9: ProgressChanged event never fires on ImageResizer?

### ISSUE THREAD

**Issue by anders9ustafsson** [briandonahue/FluxJpeg.Core#15]
_From @GoogleCodeExporter on January 20, 2016 12:59_

```
What steps will reproduce the problem?
1. instantiate an ImageResizer
2. attach a handler to the resizer's ProgressChanged event
3. Resize an image

What is the expected output? What do you see instead?
Expected progress updates via the event, but it was never thrown.


Please provide any additional information below.
Looking at the ImageResizer source, it seems like the ResizeProgressChanged
handler event is never attached/thrown, so it would never report progress.
Is this something that may come in future versions?

```

Original issue reported on code.google.com by `brian%pi...@gtempaccount.com` on 10 Aug 2009 at 9:28

_Copied from original issue: [#9](https://code.google.com/p/fjcore/issues/detail?id=9&can=1)_

**Comment by anders9ustafsson** [comment]
_From @GoogleCodeExporter on January 20, 2016 12:59_

```
FYI, this was an easy fix as well.  The ImageResizer's "ResizeProgressChanged" 
Event
Handler was never being attached to the resizeFilter.ProgressChanged event.
```

Original comment by `brian%pi...@gtempaccount.com` on 13 Aug 2009 at 4:02

**Comment by anders9ustafsson** [comment]
Has been resolved in 0.7.0.
